File: app/src/main/java/com/reddittask/android/ui/activity/main/MainView.java
package com.reddittask.android.ui.activity.main;

import com.reddittask.android.model.GameData;
import com.reddittask.android.ui.activity.base.MvpView;

/**
 * Created by marinaracu on 30.11.2017.
 */

public interface MainView extends MvpView {

    void openDetailGame(int position, String url);

    void displayGameData(GameData gameData);

    void showErrorMessage();
}

File: app/src/main/java/com/keddits/ui/activity/main/MainView.kt
package com.keddits.ui.activity.main

import com.keddits.model.Child
import com.keddits.model.GameData
import com.keddits.ui.activity.base.MvpView

/**
 * Created by marinaracu on 25.12.2017.
 */
interface MainView : MvpView {

    fun openDetailGame(url : String)

    fun displayGameData(gameData : MutableList<Child>)

    fun showErrorMessage()

    fun showLastItemLists(cildren : MutableList<Child>)
}